package.json

```diff
@@ -16,10 +16,12 @@
     "@babel/register": "^7.10.4",
     "actions-utils": "Khan/actions-utils#v1.1.0",
     "babel-eslint": "^10.1.0",
+    "chalk": "2.4.2",
```

Comment:
If chalk and prompts are only used for the tester then they should be devDependencies.